roles/template_service_broker/defaults/main.yml

```diff
@@ -3,3 +3,4 @@
 template_service_broker_remove: False
 template_service_broker_install: True
 openshift_template_service_broker_namespaces: ['openshift']
+template_service_broker_selector: '{ "role": "infra" }'
```

Comment:
To make this similar to other roles with node selectors can this be a dict object instead of a string? `{ "role": "infra" }`